lib/loader.js

```diff
@@ -41,8 +41,8 @@ module.exports = function (content) {
   this.cacheable()
   var isServer = this.options.target === 'node'
   var loaderContext = this
-  var options = this.vue || this.options.vue || {}
   var query = loaderUtils.parseQuery(this.query)
+  var options = this.options.vue || query || {}
```

Comment:
Alright, will do. I believe there was a reason I couldn't keep `this.vue`, I'll check